### ISSUE TITLE
source for pre-compiled dlls?

### ISSUE THREAD

**Issue by mjmeans** [stchan/CloneFinder#4]
Where are the sources for the precompiled dlls in your libs folder? I don't trust that these libs haven't been modified in some way.

**Comment by stchan** [comment]
The command line parser is from an ancient version of Giacomo Stellucci's project - the version used is from when the project was still hosted on codeplex. His final repo is here:

https://github.com/gsscoder/commandline

I don't know if any code from v1.8 is in that repo - you'll have to do some legwork if it isn't.

You can get the Wix dlls by installing Wix if you don't trust the assemblies in the repo.

**Comment by mjmeans** [comment]
So clicking on that link and then clicking on the link takes be to a github page that says it has been moved to a github organization page. And clicking on the on that link takes me to a full project that is also distributed as a library in NuGet which is available at: `NuGet Install CommandLineParser`. The oldest version is 1.9 `NuGet\Install-Package CommandLineParser -Version 1.9.3.15` which is contemporary to 1.8.

Wix is also available on NuGet. Like: `NuGet\Install-Package WiX -Version 3.11.2`. and it has an "EnsureWixToolsetInstalled" functionality so that other users who don't have it installed are prompted to install it.

I was going to create a PR to remove the precompiled DLLs and replace them with package references. I only had VS2017 and 2019 installed up until now, so I installed 2022 for ".NET desktop development". I got `The SDK resolver type "DotNetMSBuildSdkResolver" failed to load.` messages on all non-WiX projects.
